### PR TITLE
Manually reset background after each line

### PR DESCRIPTION
**Pull request by jima80525**
You might want to change `COLOR_RESET` to something better from colorama.  It currently only makes sense if your terminal is set to white-on-black.

### CODE REVIEW

File: blackjack/cards.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """Create Card and Deck classes as well as module variables for card printing"""
 
 from colorama import init
-from blackjack.cards_config import SUIT_SYMBOL_DICT, CARD_FACE_DICT
+from blackjack.cards_config import SUIT_SYMBOL_DICT, CARD_FACE_DICT, COLOR_RESET
```
Comment (pwildenhain) [inline blackjack/cards.py:4]: What if instead we used `Style.RESET_ALL` from `colorama`? Did you try that?

Comment (jima80525) [reply blackjack/cards.py:4]: Pushed an update to this PR with that change.  Works just fine!

**Comment by pwildenhain** [comment]
@jima80525 Thanks so much again for contributing to this! I ended up moving the card creation part of the app to a separate package (https://github.com/pwildenhain/terminal_playing_cards), but I did use this idea over there, so I'm going to merge to make sure GitHub recognizes you as a contributor on this repo :rocket:

**Comment by jima80525** [comment]
Thanks!  Super thoughtful!

**Pull request merged**